src/services/codefixes/convertTypedefToType.ts

```diff
@@ -0,0 +1,232 @@
+import {
+    Diagnostics, factory, SourceFile, SyntaxKind, Node, JSDocTypedefTag,
+    JSDocTypeLiteral, JSDocPropertyTag, JSDocTypeExpression, InterfaceDeclaration,
+    TypeAliasDeclaration, ParenthesizedTypeNode, UnionTypeNode, PropertySignature,
+    TypeNodeSyntaxKind, TypeNode, HasJSDoc, isJSDocTypedefTag, hasJSDocNodes,
+    isInJSDoc, getTokenAtPosition, textChanges, flatMap

```

Comment:
Please make these a big vertical list and sorted (may need to merge main to get linting for this); otherwise we'll end up with merge conflicts and lint errors.

Reply:
fixed